fetch.bs

```diff
@@ -1733,7 +1733,12 @@ not always relevant and might require different behavior.
 <hr>
 
 <p>A <a for=/>request</a> has an associated
-<dfn export for=request id=concept-request-priority>priority</dfn> (null or a
+<dfn export id=concept-request-priority>priority</dfn>, which is
+"<code>high</code>", "<code>low</code>" or "<code>auto</code>". Unless stated otherwise it is
+"<code>auto</code>".
+
+<p>A <a for=/>request</a> has an associated
+<dfn export for=request id=concept-request-internal-priority>internal priority</dfn> (null or a
 user-agent-defined object). Unless otherwise stated it is null.
```

Comment:
Can we make this `an <a>implementation-defined</a> object` while here? (Also applies below.)

Reply:
Done. All 3 of the user-agent-defined references have been changed.